src/SplitPane.js

```diff
@@ -269,7 +271,7 @@ class SplitPane extends React.Component {
 
     const notNullChildren = removeNullChildren(children);
 
-    const style = Object.assign(
+    let style = Object.assign(
```

Comment:
I believe you can now delete this variable declaration.

Can you also run prettier?

Reply:
Will merge after this is resolved

Reply:
Ok I did it, also removed style: styleprops ? Is this OK for you ?
run prettier & run test, it was OK